src/operator/controllers/intents_reconcilers/otterizecloud/cloud_api.go

```diff
@@ -57,9 +57,13 @@ func (c *CloudClientImpl) ReportAppliedIntents(
 
 func (c *CloudClientImpl) ReportIntentsOperatorConfiguration(
 	ctx context.Context,
-	enableEnforcement bool) error {
+	globalEnforcementEnabled bool,
+	networkPolicyEnforcementEnabled bool,
+	kafkaACLEnforcementEnabled bool) error {
```

Comment:
Please make the function receive the IntentsOperatorConfigurationInput instead, it reduces the chances that people accidentally pass the parameters in the wrong order.